README.md

```diff
@@ -2772,8 +2772,8 @@ Custom error formatters for existing and additional types can be defined with a
 
 ```ruby
 class Twitter::API < Grape::API
-  error_formatter :txt, ->(message, backtrace, options, env, original_exception) {
-    "error: #{message} from #{backtrace}"
+  error_formatter :txt, ->(message, backtrace, options, env, original_exception, status) {
```

Comment:
Is this a breaking change? Meaning if I had the old code and upgrade Grape, will this fail with an invalid number of parameters?

Reply:
Best I could tell this calls `Grape::Middleware::Error.format_message` where I defaulted status to `nil`. If someone created a custom Middleware there could be a problem. Fairpoint though.. I can check this with my own app and also look into making some more specs to verify.

Reply:
Ain't the most intuitive way but you can access the status from the `env` variable which is Rack's `env`.

You can access endpoint's status code like this `env[Grape::Env::API_ENDPOINT].status`

```ruby
describe 'error formatter env' do
  subject { last_response.body }

  let(:app) do
    Class.new(described_class) do
      format :json

      error_formatter :json, ->(message, _backtrace, _options, env, _original_exception) {
        "#{message}:#{env[Grape::Env::API_ENDPOINT].status}"
      }

      rescue_from :all do
        error!("I'm a teapot!", 418)
      end

      get { raise StandardError }
    end
  end

  before do
    get '/'
  end

  it { is_expected.to eq("I'm a teapot!:418") }
end
```

Reply:
Since we document custom middleware this would be a breaking change :(